src/libs/actions/MapboxToken.ts

```diff
@@ -1,26 +1,25 @@
-import _ from 'underscore';
 import moment from 'moment';
 import Onyx from 'react-native-onyx';
-import {AppState} from 'react-native';
-import lodashGet from 'lodash/get';
+import {AppState, NativeEventSubscription} from 'react-native';
 import ONYXKEYS from '../../ONYXKEYS';
 import * as API from '../API';
 import CONST from '../../CONST';
 import * as ActiveClientManager from '../ActiveClientManager';
+import {MapboxAccessToken, Network} from '../../types/onyx';
 
-let authToken;
+let authToken: string | undefined | null;
```

Comment:
As I see below you check for `undefined`, so this type is never acceptable.
```suggestion
let authToken: string | null;
```